contracts/examples/WitnetRNG.sol

```diff
@@ -91,8 +91,13 @@ contract WitnetRNG
         require(_queryId != 0, "WitnetRNG: not randomized");
         require(_witnetCheckResultAvailability(_queryId), "WitnetRNG: pending randomize");
         Witnet.Result memory _witnetResult = _witnetReadResult(_queryId);
-        require(witnet.isOk(_witnetResult), "WitnetRNG: failed randomize");
-        return witnet.asBytes32(_witnetResult);
+        if (witnet.isOk(_witnetResult)) {
+            return witnet.asBytes32(_witnetResult);
+        } else {
+            uint256 _nextRandomizeBlock = __randomize_[_block].nextBlock;
+            require(_nextRandomizeBlock != 0, "WitnetRNG: faulty randomize");
+            return getRandomnessAfter(_nextRandomizeBlock);
+        }
```

Comment:
Interesting.

The documentation of `getRandomnessAfter` needs to be updated as well.

Reply:
So right. Done